packages/mobile/scripts/sync_branding.sh

```diff
@@ -21,7 +21,7 @@ echo $mobile_root
 cd "$mobile_root"
 
 # Please update the sha when valora branding updates are needed
-valora_branding_sha=d5249be
+valora_branding_sha=8a74a2c
```

Comment:
For future reference, how do we get this hash?

Reply:
I think it's just from the corresponding repo under the clabs org